### PR TITLE
Task#5277_UI_data_Library

### PR DESCRIPTION
**Pull request by NguyenQuocBao19182**
50428878_2199700376948285_5336219582191370240_n
https://edu-redmine.framgia.vn/issues/5277?issue_count=18&issue_position=7&next_issue_id=5273&prev_issue_id=5278

### CODE REVIEW

**Comment by NguyenQuocBao19182** [comment]
ready

File: app/src/main/java/com/framgia/music_49/screen/library/LibraryFragment.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode, String[] permissions,
+            int[] grantResults) {
+        switch (requestCode) {
+            case MY_PERMISSIONS_REQUEST_READ_EXTERNAL_STORAGE:
+                if (grantResults[0] == PackageManager.PERMISSION_GRANTED) {
+                    // do your stuff
+                } else {
+                    Toast.makeText(getContext(), "GET_ACCOUNTS Denied", Toast.LENGTH_SHORT).show();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_49/screen/library/LibraryFragment.java]: rename lại message đi e. hoặc không cần toast luôn

**Comment by NguyenQuocBao19182** [comment]
ready

File: app/src/main/java/com/framgia/music_49/screen/library/LibraryFragment.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        } else {
+            return true;
+        }
+    }
+
+    public void showDialog(final String msg, final Context context, final String permission) {
+        AlertDialog.Builder alertBuilder = new AlertDialog.Builder(context);
+        alertBuilder.setCancelable(true);
+        alertBuilder.setTitle("Permission necessary");
+        alertBuilder.setMessage(msg + " permission is necessary");
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_49/screen/library/LibraryFragment.java]: hard code

**Comment by NguyenQuocBao19182** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**